components/tree-select/index.tsx

```diff
@@ -64,6 +64,7 @@ export interface TreeSelectProps<
   status?: InputStatus;
   switcherIcon?: SwitcherIcon | RcTreeSelectProps<ValueType, OptionType>['switcherIcon'];
   rootClassName?: string;
+  [key: `aria-${string}`]: string | boolean | number | undefined;
```

Comment:
This will lose type for each aria attribute: https://use-form.netlify.app/interfaces/_node_modules__types_react_index_d_.react.ariaattributes.html

Reply:
I have update the type definition. Thanks very much to provide me with this reference ~~